### DISCUSSION TITLE
New repository to contribute project created in many language

### DISCUSSION THREAD

**Discussion by UG-SEP** [MakeContributions/DSA#137] [Ideas]
## Hey Guys👋,
I am thinking that we should create a **new repository** where we can contribute our project `i.e. game, website, AI, console project, android app` in MakeContributions So more people can contribute there project and it will help the beginner that how to handle project management work and we also let to now about this from each other what your thought guys..
My friends had created this type repo have a look [Project-Guidance](https://github.com/Kushal997-das/Project-Guidance) and if anyone of you want to contribute fell free to contribute..
Please review it guys @ming-tsai @Pavan-Kumar122 @v007rj @Arsenic-ATG @TawfikYasser @Kushal997-das

**Comment by ming-tsai** [comment]
Hi @UG-SEP,
Ok, so we will having a list of repo?

**Comment by TawfikYasser** [comment]
I think it is a good idea @UG-SEP 

@ming-tsai I think we need only one repository and in the repo we will divide it to `project type` and the contributor will add the project to the appropriate place. What do you think!

**Comment by ming-tsai** [comment]
I prefer created a project that have a link of repos of others projects, as example of each type of project

**Reply by TawfikYasser** [reply]
You mean creating a project in the current repo and refer to the repo that contains a specific project ?

I think this a great idea too, Yes 
This will be less space and more organized and we can handle more projects over time

I am so excited to work on this type of projects and i can help more to be a maintainer `This will be a great chance for me` and I think we can make a good repo. to help new contributors.

Thanks

**Reply by Arsenic-ATG** [reply]
Yes, I like this idea too. 

It would be much better and easier to keep and maintain a list of projects than organising a repo full with projects and dependencies from different people.

**Reply by UG-SEP** [reply]
you all are right @TawfikYasser @Arsenic-ATG  and @ming-tsai

**Reply by ming-tsai** [reply]
We could use these below technologies:
- NodeJs for reading JSON file
- VueJs, React, or Angular for front-end
I'm thinking the JSON could be as below structure:
```JSON
[
    {
        "name" : "Markdown Dungeon",
        "category": "Game",
        "repoUrl": "https://github.com/MakeContributions/markdown-dungeon",
        "webUrl": "https://markdown-dungeon.netlify.app/",
        "tags": "reactjs, material-ui, gatsby, dungeon, markdown"
    }
]
```

**Reply by ming-tsai** [reply]
I know VueJs

**Reply by ming-tsai** [reply]
Or we are not going to create the front-end now?

**Comment by TawfikYasser** [comment]
Okay folks, I'll be here to start the work and organize the work among us.

**Comment by UG-SEP** [comment]
we shall start making the project and should divide the work

**Reply by ming-tsai** [reply]
I will start today about it

**Reply by UG-SEP** [reply]
> I will start today about it

ok

**Comment by ming-tsai** [comment]
Hi guys,
I have created [this project](https://github.com/MakeContributions/ideahub), welcome to contribute its

**Reply by TawfikYasser** [reply]
Really great
